repos/system_upgrade/el7toel8/actors/rpmscanner/actor.py

```diff
@@ -19,18 +20,46 @@ class RpmScanner(Actor):
     def process(self):
         output = get_installed_rpms()
 
+        pkg_repo = {}
+        header = True
+        tmp = None
+        for entry in run(['yum', 'list', 'installed'], split=True)['stdout']:
+            # Skip command output header...
+            if header:
+                if entry == 'Installed Packages':
```

Comment:
Rather try to find different way as I would afraid of translations in other languages on other machines.

Reply:
I have played a little bit with yum module. This example using returnPackages returns "installed" as repoid for all packages. The output of "yum list installed" returns "installed" for all packages installed together with system, but shows a valid repoid for packages not installed by anaconda.

Reply:
It seems there's a bug in the yum library. Even this does not work:
```python
import yum
yum_base = yum.YumBase()
for pkg in yum_base.doPackageLists().installed:
  print(pkg['repoid'])
```
It prints just 'installed'.

Reply:
Aha, found it, we shouldn't be looking at `repoid` but `ui_from_repo`, based on https://github.com/rpm-software-management/yum/blob/master/output.py#L734.
So:
```python
import yum
yum_base = yum.YumBase()
for pkg in yum_base.doPackageLists().installed:
  print(pkg['ui_from_repo'])
```
... gives us the right stuff (e.g. *@anaconda/7.6*, *@rhel-7-server-extras-rpms*, etc.)